apple-nav/src/components/Nav.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import styled from 'styled-components';
+import {NavLink} from 'react-router-dom';
+
+const StyledNav = styled(NavLink)`
```

Comment:
Good job incorporating styled components. I wasn't a believer in them at first but I'm a big fan now, the more I use them the more I like them.